.expeditor/verify.pipeline.yml

```diff
@@ -590,7 +621,7 @@ steps:
       - sudo scripts/install_golang.sh
       - sudo scripts/install_grpcurl.sh
       - cd components/compliance-service
-      - sudo -E PATH=\$(hab pkg path core/ruby)/bin:\$PATH make test-db
+      - sudo -E PATH=\$(hab pkg path core/ruby26)/bin:\$PATH make test-db
```

Comment:
InSpec is Ruby 2.7 now

Reply:
Thanks @tas50 for letting us know. We will upgrade to ruby 2.7 in the upcoming releases. This was to handle  vulnerability for an user which we  believe can be resolved by upgrading to ruby 2.6.